content/en/faq/applications/elasticsearch.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Elasticsearch Configuration
+description: Elasticsearch configuration required for use with Istio
+weight: 50
+keywords: [elasticsearch]
+---
+
+1. Elasticsearch Configuration
+
+There are two Elasticsearch configuration parameters that need to be
+set appropriately for running Elasticsearch with Istio:
```

Comment:
Why do they need to be set appropriately? What is the problem? I think the FAQ entry should start by saying something like "Depending on how it's configured, Elasticsearch may or may not work properly when running in Istio". Then explain what the problem is and that a common example is when the network.host parameter is configured to the podIP value. If instead, it's left unset (default value is 0.0.0.0), or explicitly set to 0.0.0.0 or localhost, then it should work without any further specialized configuration.

Reply:
Edited.